distribution.ini

```diff
@@ -0,0 +1,8 @@
+[Global]
+id=mozilla-flatpak
+version=1.0
+about=Mozilla Firefox Flatpak
+
+[Preferences]
+app.update.enabled=false
+browser.shell.checkDefaultBrowser=false
```

Comment:
I was under the impression that upstream did it because the default browser check also offers to *change* the default browser, which doesn't work inside a Flatpak afaik.

Reply:
Yeah, that needs a default apps portal which I think atm does not exist.

Reply:
All right, I see.